### PR TITLE
feat: README updates

### LINKED ISSUE #8: Add build/publish shield to READMEs
Once the `Makefile` for each service is "complete" (build, test, publish, etc.) we should add a shield to the README to note this so new developers/users can easily determine which services are consumable.

### PR DESCRIPTION
**Pull request by szeyan543**
This pull request enhances the README file by restructuring its content and adding comprehensive information to make it more informative and user-friendly

### CODE REVIEW

**Comment by t-fine** [comment]
@szeyan543 is this PR ready for review? I'll give it a final review and merge today, but for any other PRs you might have planned or in progress, could you do a few things to help Joe and I (or anything looking back to these?

1) A more descriptive title similar to your first commit is very helpful, something like `feat: README updates` 
2) When it comes to asking for reviews, can you use the `Reviewers` section up top on the right next to the description to request people?
3) Link the open issue to this PR in the `Development` section on the right below the reviewers thing I mentioned above.

Then I think this PR will be perfect! I created an issue to document these things when it comes to contributing guidelines. I've gone ahead and done the three on this PR

**Comment by szeyan543** [comment]
@t-fine Yes, this is ready for review. I'll make sure to provide all the things you listed next time. Thanks for the guidelines!

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Extremely simple HTTP server that responds on port 8000 with a hello message.
 
-DOCKER_HUB_ID ?= ibmosquito
+DOCKER_HUB_ID ?= szeyan11
```
Comment (t-fine) [inline Makefile]: @szeyan543 if this isn't necessary for a new version can we just revert it?

File: README.md [hunk 2]
```diff
@@ -1,75 +1,172 @@
...
-![](https://img.shields.io/github/license/open-horizon-services/web-helloworld-python)
-![](https://img.shields.io/badge/architecture-amd%2C%20amd64-green)
-![](https://img.shields.io/github/contributors/open-horizon-services/web-helloworld-python)
 
-Extremely simple HTTP server (written in Python) that responds on port 8000 with a hello message.
+![License](https://img.shields.io/github/license/open-horizon-services/web-helloworld-python)
+![Architecture](https://img.shields.io/badge/architecture-x86,arm64-green)
+![Contributors](https://img.shields.io/github/contributors/open-horizon-services/web-helloworld-python.svg)
 
-Begin by editing the variables at the top of the Makefile as desired. If you plan to push it to a Docker registery, make sure you give your docker ID. You may also want to create unique names for your **service** and **pattern** (necessary if you are sharing a tenancy with other users and you are all publishing this service).
```
Comment (t-fine) [inline README.md:8]: Can you add something about the env var `DOCKER_HUB_ID` back in?

File: README.md [hunk 3]
```diff
@@ -1,75 +1,172 @@
...
+1. **Clone the repository:**
+    Clone the `web-helloworld-python` GitHub repo from a terminal prompt on the edge node and enter the folder where the artifacts were copied.
+
+   ```shell
+   git clone https://github.com/open-horizon-services/web-helloworld-python.git
+   cd web-helloworld-python
+    ```
+
+2. **Edit Makefile:**
+    Adjust the variables at the top of the Makefile as needed, including your Docker ID and unique names for your service and pattern.
```
Comment (t-fine) [inline README.md:40]: I see you have a mention here, but maybe you could add a little block showing the values with their defaults and if they `export` them in their terminal they don't have to literally edit the values in the `Makefile` to override them

**Review by t-fine — CHANGES_REQUESTED** [review]
Some small updates

**Comment by szeyan543** [comment]
@t-fine I just modified the files according to your comments, can you check if it is okay?

**Review by t-fine — APPROVED** [review]
Looks good to me!

**Pull request merged**